source/stylesheets/_refills-styles.scss

```diff
@@ -376,32 +374,64 @@ pre[class*="language-"] {
   line-height: 1em;
 }
 
-.copy-source {
-  background-color: lighten(#333, 20%);
-  border-radius: 10px;
-  color: #fff;
-  font-size: 0.5em;
-  font-weight: 800;
-  padding: 0.4em 1em;
-  text-transform: uppercase;
-
-  &:hover,
-  &:active {
-    background-color: lighten(#333, 20%);
-    color: #fff;
-  }
-}
-
 .snippets-table {
   display: none;
 
   td {
     border-bottom: 0;
   }
+
+  .snippet:nth-child(4n) {
+    display: none
+  }
 }
 
+
 .snippet {
-  $refills-medium-screen: em(640) !default;
+  .copy-source-container {
+    min-height: 3.4em;
+
+    @include media($refills-medium-screen) {
+      min-height: 2.5em;
+    }
+
+    @include media($refills-large-screen) {
+      min-height: 1em;
+    }
+  }
+
+  &:nth-of-type(1) .copy-source {
+    background: #e98065;     
+  }
+
+  &:nth-of-type(2) .copy-source {
+    background: #55a1d4;     
+  }
+
+  &:nth-of-type(3) .copy-source:nth-of-type(1) {
+    background: #e9c05b;     
+  }
+
+  &:nth-of-type(3) .copy-source:nth-of-type(2) {
+    background: #89b981;     
+  }
+
+  .copy-source {
+    border-radius: 10px;
```

Comment:
Properties should be ordered border-bottom-right-radius, border-radius, color, display, font-size, font-weight, padding, text-transform